cassandra_init_scripts/keyspace.cql

```diff
@@ -11,9 +11,3 @@ CREATE TABLE IF NOT EXISTS stream_data (
     payload blob,
     PRIMARY KEY ((id, partition), ts, sequence_no, publisher_id, msg_chain_id)
 );
-
-create table if not exists stream_last_msg
```

Comment:
Not an accident? Cleanup of unused table?

Reply:
Yes, it was idea/experiment to speed up getting last message in the list of streams (platform), so it's just cleanup.

Reply:
👍🏻